src/compas/geometry/offset/offset.py

```diff
@@ -10,6 +10,7 @@
 from compas.geometry import centroid_points
 from compas.geometry import intersection_line_line
 from compas.geometry import normal_polygon
+from compas.geometry import is_colinear
```

Comment:
this is not necessarily related to this PR, but we need to reconsider the naming convention for this kind of functions. after i read the rule (in a PHP context) about `is_` for expected boolean returns, i read something about the need for boolean returns making sense in questions.
in this case `if points_colinear(...)` seems to make a lot more sense than `if is_colinear(...)`

Reply:
anyways...

Reply:
> in this case if points_colinear(...) seems to make a lot more sense than if is_colinear(...)

Mmm... I find `if is_colinear()` more readable than the alternative.
C# also has the convention of `is` & `has` prefixing for boolean returns, and it makes a lot of sense, it creates a very clear visual distinction of what a method returns, so clear you don't need to open the docs.